models/DataObject/ClassDefinition/Service.php

```diff
@@ -16,11 +16,12 @@
 
 namespace Pimcore\Model\DataObject\ClassDefinition;
 
-use Pimcore\Loader\ImplementationLoader\LoaderInterface;
+use Pimcore\Tool;
 use Pimcore\Logger;
 use Pimcore\Model\DataObject;
+use Pimcore\Loader\ImplementationLoader\LoaderInterface;
 use Pimcore\Model\DataObject\ClassDefinition\Data\EncryptedField;
-use Pimcore\Tool;
```

Comment:
Hi @Corepex, just a small change maybe. I think it would be better if we just stick to the alphabetical order of use statements.

Reply:
reordered use statements in alphabetical order